rootfs/etc/os-release

```diff
@@ -0,0 +1,9 @@
+NAME="Chimera OS"
```

Comment:
NAME and PRETTY_NAME should probably just be "ChimeraOS"

Reply:
Arch Linux uses Arch Linux in both fields.

Reply:
> Arch Linux uses Arch Linux in both fields.

Yeah, that makes sense since they don't have named releases. I'm just saying this could help us on identifying the specific release. But If there is no objection, using `ChimeraOS` for both should be fine

Reply:
From the definition it sounds like @Samsagax is right and `NAME` should be `ChimeraOS` and `PRETTY_NAME` should be `ChimeraOS <RELEASE NUMBER>`

Reply:
Someone will have to remember to replace the release number every time. Should I set the number to 27?

Reply:
You need to modify the build.sh script to replace a variable inside the file at build time. Maybe the best way is to plug that on the postinstallhook but I would prefer if the creation is done at the same time as the replacement.

Reply:
I think the easiest and best thing to do is to generate the file in `build.sh` similar to how we generate `/etc/lsb_release`.
See here: https://github.com/ChimeraOS/chimeraos/blob/master/build.sh#L172